packages/api/src/controllers/user.ts

```diff
@@ -1477,4 +1500,51 @@ app.post(
   }
 );
 
+app.post(
+  "/migrate/userDefaultProject",
```

Comment:
When do we plan to call it? Do we plan to do some global migration of the users? Is it something we'll do once and then remove this endpoint? 

I think you can add some comment to explain that.

Reply:
We plan to call it on production deploy - it's done once and then removed
Sure can add some comments